styles.css

```diff
@@ -0,0 +1,163 @@
+body {
+	font-family: 'Old Standard TT', serif;
+	line-height: 1.4;
+	margin: 0;
+	background: rgba(0, 214, 170, .9);
+	font-weight: 400;
+}
+
+.container {
+	max-width: 960px;
+	margin: auto;
+	overflow: hidden;
+	padding: 0 30px;
+}
+
+.showcase {
+	background-color: rgba(130, 149, 166, .75);
+	color: white;
+	height: 100vh;
+	position: relative;
+}
+
+.showcase::before {
+	content: '';
+	background: url(wallpaper.jpg) no-repeat center center/cover;
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	z-index: -1;
+}
+
+.showcase .showcase-inner {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	text-align: center;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/burger-one/pullRequest?prid=6050239)